catalog/spatial/csw/spatial-csw-source-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/source/CswFilterDelegate.java

```diff
@@ -59,6 +60,17 @@
 
   private static final Logger LOGGER = LoggerFactory.getLogger(CswFilterDelegate.class);
 
+  private static final String ATTEMPTING_TO_BUILD_MSG =
+      "Attempting to build {} filter for property {} and WKT {} in LON/LAT order.";
+
+  private static final String CSW_SUPPORT_EXCEPTION_MSG =
+      "CSW source does not support {} filter or any of its fallback spatial filters. This may be due to spatial operators not being supported or geometry operands not being supported.  See the Get Capabilities Response to determine the cause.";
+
+  private static final String CSW_SUPPORT_EXCEPTION_MSG_SIMPLE =
```

Comment:
❔ Should _SIMPLE be `CSW source does not support`?

Reply:
Yup! good catch